src/util.py

```diff
@@ -11,4 +12,27 @@ def update_config(config,args,exclude=[],required=[]):
         if (config.get(param) is None):
             raise ValueError("parameter {0} unspecified. Please provide a value via the command line or in the config file.".format(param))
 
-    return config
\ No newline at end of file
+    return config
+    
+def recursively_find_files(resource_name):
```

Comment:
Excellent, so let' just add a small comment in the code. Do you want also the method to throw an exception when it's called with a file in lieu of a folder?